Measurements/Workbook_Measurements.ipynb

```diff
@@ -450,7 +450,7 @@
     " - all the bit strings in the second array have the same value in this position (all `bits2[j][Q]` are the same),\n",
     " - these values are different for the first and the second arrays.\n",
     "\n",
-    "> For example, for arrays `[[false, true, false], [false, true, true]]` and `[[true, false, true], [false, false, true]]` return 0 corresponds to state $\\frac{1}{\\sqrt2}\\big(|010\\rangle + |011\\rangle\\big)$, return 1 - to state $\\frac{1}{\\sqrt2}\\big(|101\\rangle + |001\\rangle\\big)$, and you can distinguish these states perfectly by measuring the second qubit.\n",
+    "> For example, for arrays `[[false, true, false], [false, true, true]]` and `[[true, false, true], [false, false, true]]` return 0 corresponds to state $\\frac{1}{\\sqrt2}\\big(|010\\rangle + |011\\rangle\\big)$, return 1 to state $\\frac{1}{\\sqrt2}\\big(|101\\rangle + |001\\rangle\\big)$, and you can distinguish these states perfectly by measuring the second qubit.\n",
```

Comment:
I think we should keep the `-`, since it indicates that the word "corresponded" was omitted?

Reply:
Again, as a layman :-), might prefer not to have symbols among words, if avoidable.
Tend to look for mathematical relevance/meaning, especially when it follows a number.

Reply:
Then let's say "and return 1 corresponds to state ...", so that there is no word omission. 

Can you please do these two changes?

Reply:
Not sure if it's the correct approach, but created a fresh pull request with the suggested changes

Reply:
Typically you edit the branch from which you sent this pull request to apply any requested changes on top of the changes you proposed originally, so that in the end they all get merged together. Right now, for example, this PR contains a change not included in the other one (the escape character before % symbol on line 475). 

Looks like you've already deleted the original branch from which this PR was sent, so let's move over to #577 and apply matching changes there. I'll close this PR

Reply:
I'll keep that in mind and wait before I delete forked repositories in future.
Thanks for explaining

Reply:
Yes, it's better to keep the repo (and the branch in it) intact until the fate of the pull request is resolved completely :-)

No worries, it's all good things to learn.